### PR TITLE
Make quantity field on StockInput required

### PR DESCRIPTION
**Pull request by IKarbowiak**
Field `quantity` on `StockInput` should be required.

# Impact

* [ ] New migrations
* [x] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [x] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1379 API crashes when passing invalid integer as stock quantity](https://app.clickup.com/t/2549495/SALEOR-1379)

**Comment by d-wysocki** [comment]
Maybe we should add test for this required field? To be sure that we need to pass this value in mutation?

**Review by maarcingebala — APPROVED** [review]

**Comment by IKarbowiak** [comment]
> Maybe we should add test for this required field? To be sure that we need to pass this value in mutation.

As far I know, we don't make tests for GraphQL validations.

**Review by d-wysocki — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fix invalid tax rates for lines - #7058 by @IKarbowiak
 - Allow seeing unconfirmed orders - #7072 by @IKarbowiak
 - Raise GraphQLError when too big integer value is provided - #7076 by @IKarbowiak
+- Make quantity field on StockInput required - #7082 by @IKarbowiak
```
Comment (korycins) [inline CHANGELOG.md:39]: Question - Shouldn't be in breaking changes section? Assuming that you have an App that does something with Stock over API.

**Pull request merged**